packages/dialtone-vue3/components/toast/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as DtToast } from './toast.vue';
+export { default as DtToastLayoutAlternateIcon } from './layouts/toast_layout_alternate_icon.vue';
```

Comment:
No need to export it here, as it is not meant to be used independently, it is an internal component. 

```suggestion
```